python/ray/experimental/signal.py

```diff
@@ -7,6 +7,9 @@
 import ray
 import ray.cloudpickle as cloudpickle
 
+# This string should be identical to the name of the signal sent upon
+# detecting that an actor died.
+ActorDiedStr = "ACTOR_DIED"
```

Comment:
1) explain in the comment where this constant in used on the c++ side.
2) constant name should use uppercase letters, e.g., `ACTOR_DIED_SIGNAL`.